config.py

```
@@ -60,14 +63,101 @@ def _detect_system_memory():
+    except (ValueError, TypeError):
+        print(f"Warning: Invalid {var_name} value, using default {default}")
+        return default
+
+# Load thresholds from environment variables with fallback to defaults
+TAU_FIELD_ACCEPT  = _get_float_env_var("TAU_FIELD_ACCEPT", 0.95)   # auto-accept threshold
+TAU_ENHANCER_PASS = _get_float_env_var("TAU_ENHANCER_PASS", 0.90)  # enhanced processing threshold  
+TAU_LLM_PASS      = _get_float_env_var("TAU_LLM_PASS", 0.85)       # LLM fallback threshold
+
+# Validate threshold ordering (should be descending)
+if not (TAU_LLM_PASS <= TAU_ENHANCER_PASS <= TAU_FIELD_ACCEPT):
+    print(f"Warning: Threshold ordering is incorrect. Expected TAU_LLM_PASS ≤ TAU_ENHANCER_PASS ≤ TAU_FIELD_ACCEPT")
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)